docs/references/platform-sdk.md

```diff
@@ -0,0 +1,94 @@
+# Platform SDK
+
+The Platform SDK is a modern superset of the capabilities available in
+the former [Engine and HTML GUI
+SDKs](https://help.alteryx.com/developer-help/legacy-sdks). Previous SDKs use outdated technology and limit
+extension opportunities. The new SDKs use the latest open-source
+technology and open up new ways to extend Alteryx products.
+
+### Overview
+
+The Platform SDK lets you extend your Alteryx experience by building
+custom tools and plugins.
+
+While Alteryx Designer provides a wide range of functionality with the
+available tools, you might find that you perform a specific action or
+task that could be better served by creating a custom tool or plugin.
+You can distribute custom tools within your organization and environment
+and still leverage the speed and flexibility of Designer and Server.
+
+A custom tool or plugin consists of 2 components:
+
+1.  The user interface, shown in Designer's Configuration window. To
+    create the user interface component, use the [AYX UI
+    SDK](https://alteryx.github.io/alteryx-ui/)
+    (a React-based front-end SDK that lets you build custom applications
+    to use within the Alteryx platform).
+2.  The engine, which processes logic to handle records passed to and
+    from the Alteryx Engine. To create the engine component, use the
+    [AYX Python
+    SDK](https://help.alteryx.com/developer-help/ayx-python-sdk).
+
+To help facilitate the creation of a custom tool or plugin, you can
+leverage the [AYX Plugin
+CLI](https://help.alteryx.com/developer-help/ayx-plugin-cli).
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version